dockers/docker-bgp/minigraph_facts.py

```diff
@@ -0,0 +1,432 @@
+#!/usr/bin/env python
```

Comment:
for 1, we can move it do the docker-base and stablized there.

for 2, configuration generate should be part of the docker container's job. For example, if we switch quagga to gobgp, the configuration template for them is going to be very different, where should be store the config template? I feel should be stored in the container.